README.md

```diff
@@ -36,20 +37,18 @@ are:
     phabricator      Fetch tasks from a Phabricator site
     pipermail        Fetch messages from a Pipermail archiver
     redmine          Fetch issues from a Redmine server
-    rocketchat       Fetch messages from a Rocket.Chat channel   
+    rocketchat       Fetch messages from a Rocket.Chat channel
     rss              Fetch entries from a RSS feed server
     slack            Fetch messages from a Slack channel
     stackexchange    Fetch questions from StackExchange sites
     supybot          Fetch messages from Supybot log files
     telegram         Fetch messages from the Telegram server
-    twitter          Fetch tweets from the Twitter Search API
```

Comment:
You need to revert this change, perceval support exist for twitter backend.

Reply:
I also see that info about twitter backend doesn't show for `perceval --help`. This might be because it was not added to the script. Can you please add the same line here too?
https://github.com/chaoss/grimoirelab-perceval/blob/master/bin/perceval#L83